src/main/java/hudson/security/PAMSecurityRealm.java

```diff
@@ -58,6 +59,7 @@
  * @author Kohsuke Kawaguchi
  * @since 1.282
  */
+@SuppressRestrictedWarnings(Messages.class)
```

Comment:
Nope, this is a real problem that got caught for you by updating! `jenkinsci/jenkins/core/src/main/resources/hudson/security/Messages.properties` is still defining PAM-related keys (guess these were forgotten when splitting the plugin out of core). They need to be copied to a new `Messages.properties` in this plugin _in a different package_ unique to this plugin, and then sometime later deleted from core. See https://github.com/jenkinsci/junit-plugin/pull/103 for example, or https://github.com/jenkinsci/mailer-plugin/pull/28.

Reply:
Reference: https://github.com/jenkinsci/jenkins/pull/2656

Reply:
Thanks, didn't know about that. Updated.